tests/unit/common/test_validation_scheduler.py

```diff
@@ -0,0 +1,100 @@
+import uuid
+import json
+from unittest.mock import patch
+
+from upload.common.upload_area import UploadArea
+from upload.common.uploaded_file import UploadedFile
+from upload.common.validation_scheduler import ValidationScheduler, MAX_FILE_SIZE_IN_BYTES
+from upload.common.database import UploadDB
+
+from .. import UploadTestCaseUsingMockAWS
+
+
+class TestValidationScheduler(UploadTestCaseUsingMockAWS):
+
+    def setUp(self):
+        super().setUp()
+
+        self.upload_area_id = str(uuid.uuid4())
+        self.upload_area = UploadArea(self.upload_area_id)
+        self.upload_area.update_or_create()
+
+    def tearDown(self):
+        super().tearDown()
+        pass
+
+    @patch('upload.common.upload_area.UploadedFile.size', MAX_FILE_SIZE_IN_BYTES + 1)
+    def test_check_file_can_be_validated_returns_false_if_file_is_too_large_for_validation(self):
```

Comment:
please start using my double-underscore technique:

`def test_<thing_under_test>__<scenario>__<should_do>`
e.g.
`def test_check_file_can_be_validated__when_file_is_too_large_for_validation__returns_false`

Reply:
I guess this is just a move of code, but it's a good time to revise it.

Reply:
This is not a technique that works everywhere, but it's good here.

Reply:
fixed all tests to follow this pattern in test_validation_scheduler.py